### PR TITLE
adds a basic e2e test for --dbt using duckdb

### PR DESCRIPTION
**Pull request by dlawin**
Relates to #418 

Runs a --dbt diff against a duckdb database in the repo and does some basic assertions

### CODE REVIEW

File: tests/dbt_artifacts/jaffle_shop.duckdb
Comment (dlawin) [inline tests/dbt_artifacts/jaffle_shop.duckdb:1]: 4.51 MB is larger than I expected

File: tests/test_dbt.py [hunk 1]
```diff
@@ -334,16 +334,18 @@ def test_get_connection_no_type(self, mock_open):
...
-            assert diff[-1].decode("utf-8") == "Diffs Complete!"
-        else:
-            pass
+        artifacts_path = os.getcwd() + '/tests/dbt_artifacts'
+        test_project_path = os.environ.get("DATA_DIFF_DBT_PROJ") or artifacts_path
+        diff = run_datadiff_cli("--dbt", "--dbt-project-dir", test_project_path, "--dbt-profiles-dir", test_project_path)
+        assert diff[-1].decode("utf-8") == "Diffs Complete!"
+
+        # assertions for the diff that exists in tests/dbt_artifacts/jaffle_shop.duckdb
+        if test_project_path == artifacts_path:
```
Comment (dlawin) [inline tests/test_dbt.py:345]: These asserts are pretty random at the moment. If we add json output support these could be improved considerably

File: .gitignore [hunk 2]
```diff
@@ -74,6 +74,9 @@ docs/_build/
 # PyBuilder
 target/
 
+# Exception for dbt tests
+!tests/dbt_artifacts/target
```
Comment (williebsweet) [inline .gitignore:78]: Why include the test target info? For debugging?

File: tests/dbt_artifacts/jaffle_shop.duckdb
Comment (williebsweet) [reply tests/dbt_artifacts/jaffle_shop.duckdb:1]: Whats the cost in terms of time? Is that the primary concern?

Comment (dlawin) [reply tests/dbt_artifacts/jaffle_shop.duckdb:1]: Ultimately I think we are only negatively impacting git clone download speed. Pretty minor concern compared to preventing regressions.

**Review by williebsweet — APPROVED** [review]

**Pull request merged**